ui/app/components/sidebar/user-menu.hbs

```diff
@@ -44,7 +44,7 @@
                   @text="Copy token"
                   @textToCopy={{this.auth.currentToken}}
                   @isFullWidth={{true}}
-                  class="hds-copy-button size-7 hidden-icon link is-flex-start"
+                  class="hds-copy-button in-dropdown link is-flex-start"
```

Comment:
Combined 2 classes into one, this is applied to copy buttons which are in dropdowns